src/main/java/edu/tamu/scholars/middleware/discovery/controller/IndexingController.java

```diff
@@ -1,29 +1,29 @@
-package edu.tamu.scholars.middleware.discovery.controller;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
-
-import edu.tamu.scholars.middleware.discovery.service.IndexService;
-
-@RestController
-@RequestMapping("/index")
-public class IndexingController {
-
-    @Autowired
-    private IndexService indexService;
-
-    @GetMapping("/isIndexing")
-    public ResponseEntity<Boolean> isIndexing() {
-        Boolean isIndexing = indexService.isIndexing();
-
-        return isIndexing
-            ? ResponseEntity.status(HttpStatus.TOO_EARLY)
-                .body(isIndexing)
-            : ResponseEntity.ok(isIndexing);
-    }
-
-}
+package edu.tamu.scholars.middleware.discovery.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.tamu.scholars.middleware.discovery.service.IndexService;
+
+@RestController
+@RequestMapping("/index")
+public class IndexingController {
```

Comment:
Is there no change here?

Reply:
Whitespace changes.

Reply:
See PR description `Whitespace cleanup for consistent LF line endings`. You can turn off whitespace changes in files view.

https://github.com/TAMULib/scholars-discovery/pull/239/files?diff=split&w=1

Reply:
<img width="1473" alt="image" src="https://user-images.githubusercontent.com/8352733/214171199-54a12126-f5bc-49bc-a9bc-09818bc10fef.png">